docs/content/en/integrations/ldap-authentication.md

```diff
@@ -0,0 +1,129 @@
+---
+title: "Authentication via LDAP"
+description: "Authenticate users using LDAP"
+draft: false
+weight: 3
```

Comment:
```suggestion
weight: 4
```

Although it is great to have a documentation for LDAP integration, it should come behind the integration for OAuth2 authentication, because this is more widely used these days.